packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -391,6 +398,17 @@ export default function DimensionsPanel( {
 	};
 	const hasMinHeightValue = () => !! value?.dimensions?.minHeight;
 
+	// Height
+	const showHeightControl = useHasHeight( settings );
+	const heightValue = decodeValue( inheritedValue?.dimensions?.height );
+	const setHeightValue = ( newValue ) => {
+		onChange( setImmutably( value, [ 'dimensions', 'height' ], newValue ) );
```

Comment:
Not sure if this is an inconsistency with the changes to dimensions.php in 66c1d6a592f51a26a47e58343c5ea24aa155519f, but the UI component doesn't clear aspect ratio when height is set (not a blocker). Should it? 🤷🏻 

This is a UX polish item - the backend correctly handles the conflicts. Can be a follow up.